### PR TITLE
IWF-105: Add IwfEvent definition

### PR DESCRIPTION
**Pull request by lwolczynski**

### CODE REVIEW

File: iwf.yaml [hunk 1]
```diff
@@ -1476,3 +1476,30 @@ components:
           format: int64
         attempt: # it's from Attempt of Cadence/Temporal activity.GetInfo
           type: integer
+    EventType:
+      type: string
+      enum:
+        - WORKFLOW_START_EVENT
+        - WORKFLOW_COMPLETE_EVENT
+        - SET_DATA_ATTRIBUTES
```
Comment (longquanzheng) [inline iwf.yaml]: Oh sorry I made this up in the doc for discussion. We don’t need them yet.
But we need other events like state api start, state api complete, state api execution attempt  etc

Comment (lwolczynski) [reply iwf.yaml]: I want to add even a subset of them, for now, to be able to use this new type, `IwfEvent`. Let's discuss what we need

File: iwf.yaml [hunk 2]
```diff
@@ -1476,3 +1476,29 @@ components:
...
+        workflowType:
+          type: string
+        workflowId:
+          type: string
+        workflowRunId:
+          type: string
+        stateId:
+          type: string # only applicable for certain events
+        stateExecutionId:
+          type: string # only applicable for certain events
```
Comment (longquanzheng) [inline iwf.yaml]: Maybe also need startTime and endTime(not applicable for start event)?

Comment (longquanzheng) [reply iwf.yaml]: we can then use the endTime to calculate the duration

Comment (lwolczynski) [reply iwf.yaml]: As time (string) or timestamp (int64)? I'm thinking the latter

Comment (longquanzheng) [reply iwf.yaml]: Yeah int64 would be great

Comment (lwolczynski) [reply iwf.yaml]: Done

**Review by longquanzheng — APPROVED** [review]

**Pull request merged**